Extensions/render/fonts.lisp

```diff
@@ -56,7 +74,7 @@
 (declaim (inline gcache-get))
 
 (defun gcache-get (cache key-number)  
-  (declare (optimize (speed 3))
+  (declare ;;(optimize (speed 3))
            (type (simple-array t (512))))
```

Comment:
why did you remove speed delcaration here? if it's bogus (i.e break things, either reduce it to lower level, or remove)